draft-ietf-quic-invariants.md

```diff
@@ -159,21 +161,13 @@ All other bits in that byte are version specific.
 
 The next four bytes include a 32-bit Version field (see {{version}}).
 
-The next byte contains the length in bytes of the two Connection IDs (see
-{{connection-id}}) that follow.  Each length is encoded as a 4-bit unsigned
-integer.  The length of the Destination Connection ID (DCIL) occupies the high
-bits of the byte and the length of the Source Connection ID (SCIL) occupies the
-low bits of the byte.  An encoded length of 0 indicates that the connection ID
-is also 0 bytes in length.  Non-zero encoded lengths are increased by 3 to get
-the full length of the connection ID; the final value is therefore either 0 or
-between 4 and 18 bytes in length (inclusive).  For example, an byte with the
-value 0xe0 describes a 17 byte Destination Connection ID and a zero byte Source
-Connection ID.
-
-The connection ID lengths are followed by two connection IDs.  The connection
-ID associated with the recipient of the packet (the Destination Connection ID)
-is followed by the connection ID associated with the sender of the packet (the
-Source Connection ID).
+The next byte contains the length in bytes of the destination Connection ID
+(see {{connection-id}}) that follows.  The length is encoded as an 8-bit
+unsigned integer. This is followed by the destination Connection ID itself,
+associated with the receiver of the packet.
+
+The next two fields are the Connection ID length for the source and the source
```

Comment:
```suggestion
The next two fields are the length in bytes of the source Connection ID and the source
```

Reply:
I am happy to commit this, but it's 'outdated'. Can you update?